package.json

```diff
@@ -76,6 +76,7 @@
     "better-docs": "^1.4.7",
     "husky": "^4.2.3",
     "jest-dom": "^3.5.0",
+    "jsdoc": "^3.6.4",
```

Comment:
Thats strange. This is already on master. Can you rebase?

Reply:
And remove the changes from the package-json.lock

Reply:
i think the problem is that I used it as a dev dependency but it was already installed as non-dev dependency. i think it should remain dev

Reply:
Oh I thought this had be moved to dev. Yeh you can leave this one and remove the one from `dependencies` that'd be great.